tests/director/external-services-mock-integration/fixtures_queries.go

```diff
@@ -14,3 +18,66 @@ func unregisterApplicationInTenant(t *testing.T, ctx context.Context, gqlClient
 	req := fixDeleteApplicationRequest(t, id)
 	require.NoError(t, tc.RunOperationWithCustomTenant(ctx, gqlClient, tenant, req, nil))
 }
+
+func registerApplication(t *testing.T, ctx context.Context, gqlClient *gcli.Client, name, tenant string) graphql.ApplicationExt {
```

Comment:
As this fixes seem to be copied from API tests, I think they should be moved to different package and imported in both API and this tests.